ament_cppcheck/bin/ament_cppcheck

```diff
@@ -41,7 +42,11 @@ def main(argv=sys.argv[1:]):
         print('No files found', file=sys.stderr)
         return 1
 
-    cppcheck_bin = find_executable('cppcheck')
+    additional_paths = None
+    if os.name == 'nt':
+        # search in location where cppcheck is installed via chocolatey
+        additional_paths = ['C:/Program Files (x86)/Cppcheck']
```

Comment:
Please use `%ProgramFiles%` instead. `Program Files` is mapped to a localized path in non-English installations of Windows:

http://en.wikipedia.org/wiki/Program_Files